pandas/core/arrays/base.py

```diff
@@ -365,7 +367,7 @@ def isna(self):
         raise AbstractMethodError(self)
 
     def _values_for_argsort(self):
-        # type: () -> ndarray
+        # type: () -> np.ndarray
```

Comment:
we can switch to the py3 style now i think (here or later)

Reply:
Yea planning to do later to minimize diff

Reply:
great